src/service/viewer-impl.js

```diff
@@ -890,7 +890,12 @@ export class Viewer {
   isTrustedViewerOrigin_(urlString) {
     /** @const {!Location} */
     const url = parseUrlDeprecated(urlString);
-    if (url.protocol != 'https:') {
+    const {protocol} = url;
+    // Android x-thread is allowed.
```

Comment:
Nit: Both Android and iOS will use this. How about `Mobile WebView ...`?

Reply:
done.